Comp2022/src/main/include/commands/AutoShootDriveShoot.h

```diff
@@ -0,0 +1,44 @@
+// RobotBuilder Version: 3.1
+//
+// This file was generated by RobotBuilder. It contains sections of
+// code that are automatically generated and assigned by robotbuilder.
+// These sections will be updated in the future when you export to
+// C++ from RobotBuilder. Do not put any code or make any change in
+// the blocks indicating autogenerated code or it will be lost on an
+// update. Deleting the comments indicating the section will prevent
+// it from being updated in the future.
+
+#pragma once
+
+#include "subsystems/Drivetrain.h"
+#include "subsystems/FloorConveyor.h"
+#include "subsystems/Intake.h"
+#include "subsystems/Shooter.h"
+#include "subsystems/VerticalConveyor.h"
+
+#include <frc2/command/CommandBase.h>
+#include <frc2/command/CommandHelper.h>
+#include <frc2/command/SequentialCommandGroup.h>
+
+/**
+ *
+ *
+ * @author ExampleAuthor

```

Comment:
Can you link to the slides in the slideshow that shows what the routine does? https://docs.google.com/presentation/d/1SQlcOZpuv9mqVmsuJRcJQMyKcwuBd6hcQGlkFT-_Odg/edit#slide=id.g11030b7b884_1_28 AFAIKT this is the slide for this routine

Reply:
If we had the same programming mentor in every day to work on this with you, I wouldn't ask for this level of detail in documentation - but because it's a mix of Connor, Andrew, and Jeff on different days, it really helps us all stay on the same page if you invest some extra cycles in really good documentation

Reply:
Done!